Add documentation for changes in the media player package to support new media schemas (#588)
snowplow/documentation: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/media-player/index.md

Reviewer: The context names for redshift no longer require the `source(...` stuff, can you update the default value please. Also maybe call this out in the migration guide/as breaking changes in the changelog.

Author: Changed and added a mention in migration guide.

| `max_media_pv_window` | The number of hours that needs to pass before new page_view level media player metrics from the `snowplow_media_player_base` table are safe to be processed by the model downstream in the `snowplow_media_player_media_stats` table. Please note that even if new events are added later on ( e.g. new `percentprogress` events are fired indicating potential replay) and the `snowplow_media_player_base` table is changed, the model will not update them in the media_stats table, therefore it is safer to set as big of a number as still convenient for analysis and reporting. | 10 |
| `valid_play_sec` | The minimum number of seconds that a media play needs to last to consider that interaction a valid play. The default is 30 seconds (based on the YouTube standard) but it can be modified here, if needed. | 30 |
| `surrogate_key_treat_nulls_as_empty_strings` | Passed through to `dbt_utils` to match legacy surrogate key behavior. | `true` |

### Contexts, filters, and logs

| Variable Name | Description | Default |
| --- | --- | --- |

> | `enable_whatwg_media` | Set to `true` if the HTML5 video element context schema is enabled. This variable is used to handle syntax depending on whether the context fields are available in the database or not. | `false` |

Reviewer 2: ```suggestion
| `enable_whatwg_media` | Set to `true` if the [HTML5 media element context schema](https://github.com/snowplow/iglu-central/blob/master/schemas/org.whatwg/media_element/jsonschema/1-0-0) is enabled. This variable is used to handle syntax depending on whether the context fields are available in the database or not. | `false` |
```

> | `enable_whatwg_video` | Set to `true` if the HTML5 video element context schema is enabled. This variable is used to handle syntax depending on whether the context fields are available in the database or not. | `false` |

Reviewer 2: ```suggestion
| `enable_whatwg_video` | Set to `true` if the [HTML5 video element context schema](https://github.com/snowplow/iglu-central/blob/master/schemas/org.whatwg/video_element/jsonschema/1-0-0) is enabled. This variable is used to handle syntax depending on whether the context fields are available in the database or not. | `false` |
```

> | `enable_youtube` | Set to `true` if the YouTube context schema is enabled. This variable is used to handle syntax depending on whether the context fields are available in the database or not. | `false` |

Reviewer 2: ```suggestion
| `enable_youtube` | Set to `true` if the [YouTube context schema](https://github.com/snowplow/iglu-central/blob/master/schemas/com.youtube/youtube/jsonschema/1-0-0) is enabled. This variable is used to handle syntax depending on whether the context fields are available in the database or not. | `false` |
```
